src/api/interactivity.js

```diff
@@ -139,8 +139,11 @@ export default class Interactivity {
     _init(layerList) {
         this._emitter = mitt();
         this._layerList = layerList;
-        this._subscribeToIntegratorEvents(layerList[0].getIntegrator());
         this._prevFeatures = [];
+        Promise.all(layerList.map(layer => layer._context)).then(() => {
```

Comment:
I think we need some kind of public method `isLoaded` instead of relying on private properties.

What happens if in the future we need the context and `metadatum` to consider a layer loaded?

Reply:
"What happens if in the future we need the context and metadatum to consider a layer loaded?" => "If in the future" is a smell already